drivers/counter/counter_ll_stm32_rtc.c

```diff
@@ -129,9 +132,8 @@ static int rtc_stm32_set_alarm(struct device *dev, u8_t chan_id,
 
 	data->callback = alarm_cfg->callback;
 	data->user_data = alarm_cfg->user_data;
-	data->absolute = alarm_cfg->absolute;
 
-	if (!alarm_cfg->absolute) {
+	if (!(alarm_cfg->absolute & COUNTER_ALARM_FLAG_ABSOLUTE)) {
```

Comment:
`alarm_cfg->flags`
more occurrences follow

Reply:
fixed. I have also updated test_counter.c and mian.c (sample/drivers/counter)